acceptance-tests/src/main/java/it/com/atlassian/bitbucket/jenkins/internal/applink/oauth/client/JenkinsApplinksClient.java

```diff
@@ -16,18 +20,20 @@
     private static final String CREATE_OAUTH_CONSUMER_PATH = "/bbs-oauth/create/performCreate";
 
     private final ObjectMapper jsonSerializer = new ObjectMapper();
-    private final String baseUrl;
+    private final JenkinsUtils jenkinsUtils;
 
     public JenkinsApplinksClient(String baseUrl) {
-        this.baseUrl = removeEnd(baseUrl, "/");
+        jenkinsUtils = new JenkinsUtils(removeEnd(baseUrl, "/"));
```

Comment:
you change from `removeStart` to `removeEnd` I assume there is a reason for this? `ermoveEnd` seems more correct to me, but I'm curious.